test/kubernetes_unittest.cc

```diff
@@ -137,4 +147,77 @@ TEST_F(KubernetesTest, ComputePodAssociations) {
       ComputePodAssociations(reader, pod->As<json::Object>());
   EXPECT_EQ(expected_associations->ToString(), associations->ToString());
 }
+
+TEST_F(KubernetesTest, GetContainerMetadata) {
+  Configuration config(std::stringstream(
+    "KubernetesClusterName: TestClusterName\n"
+    "KubernetesClusterLocation: TestClusterLocation\n"
+    "MetadataApiResourceTypeSeparator: \".\"\n"
+  ));
+  Environment environment(config);
+  KubernetesReader reader(config, nullptr);  // Don't need HealthChecker.
+  json::value pod = json::object({
+    {"metadata", json::object({
+      {"namespace", json::string("TestNamespace")},
+      {"name", json::string("TestName")},
+      {"uid", json::string("TestUid")},
+      {"creationTimestamp", json::string("2018-03-03T01:23:45.678901234Z")},
+      {"labels", json::object({{"label", json::string("TestLabel")}})},
+    })},
+  });
+  json::value spec = json::object({{"name", json::string("TestSpecName")}});
+  json::value status = json::object({{
+    "containerID", json::string("docker://TestContainerID")}});
```

Comment:
When split over multiple lines, let's use the following formatting:
```cpp
  json::value status = json::object({
    {"containerID", json::string("docker://TestContainerID")},
  });
```

Reply:
Fixed